Gifski/util.swift

```diff
@@ -236,7 +236,36 @@ extension NSWindowController: NSWindowDelegate {
 }
 
 
+extension NSView {
+	private final class AddedToSuperviewObserverView: NSView {
+		var onAdded: (() -> Void)?
 
+		override var acceptsFirstResponder: Bool {
+			return false
+		}
+
+		convenience init() {
+			self.init(frame: .zero)
+		}
+
+		override func viewDidMoveToWindow() {
+			guard window != nil else {
+				return
+			}
+
+			onAdded?()
+			removeFromSuperview()
+		}
+	}
+
+	// Enables you do add contraints and do other initialization without having to subclass the view.
+	// TODO: Please show me a better way to achieve this than using an invisible view 🙏
+	func onAddedToSuperview(_ closure: @escaping () -> Void) {
```

Comment:
Does anyone know a better way of setting up constraints to a superview without having to subclass the view and add those contraints in `loadView`?

Reply:
Can’t you add the constraints after you call `addSubview’